### PR TITLE
Updated Jenkinsfile and shell scripts, some CRD configs

### PR DESCRIPTION
**Pull request by nathan-weinberg**
This PR makes the following changes:
- Simplifies Jenkinsfile with additional template directions and scripting moved to the shell scripts
- Removed need for `$WORKSPACE` variable if running `netobserv.sh` scripts locally
- Updated shell scripts with additional logging and minor functionality changes
- Added `nukeobserv` function to completely remove all NetObserv artifacts from a cluster
- Updated CRDs by changing `cacheMaxFlows` value for eBFP
- `uninstall_netobserv` no longer requires `INSTALLATION_SOURCE` to be set
- Switched `Setup FLP service and service-monitor` and `Configure flowcollector and Kafka` Jenkins stage positions and names
- Minor fix to NOPE tool

### CODE REVIEW

**Review by memodi — CHANGES_REQUESTED** [review]

File: scripts/flows_v1alpha1_flowcollector_lokistack.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ spec:
       imagePullPolicy: Always
       sampling: 1
       cacheActiveTimeout: 5s
-      cacheMaxFlows: 1000
+      cacheMaxFlows: 100000
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply scripts/flows_v1alpha1_flowcollector_lokistack.yaml:25]: For sure - for both files or just this one?

File: scripts/nope.py [hunk 2]
```diff
@@ -415,8 +415,8 @@ def main():
         logging.error("START_TIME and END_TIME are needed to proceed")
         sys.exit(1)
     else:
-        logging.info("Parsed Start Time: " + datetime.datetime.fromtimestamp(int(START_TIME)).strftime('%I:%M%p%Z UTC on %m/%d/%Y'))
-        logging.info("Parsed End Time:   " + datetime.datetime.fromtimestamp(int(END_TIME)).strftime('%I:%M%p%Z UTC on %m/%d/%Y'))
+        logging.info("Parsed Start Time: " + datetime.datetime.utcfromtimestamp(int(START_TIME)).strftime('%I:%M%p%Z UTC on %m/%d/%Y'))
+        logging.info("Parsed End Time:   " + datetime.datetime.utcfromtimestamp(int(END_TIME)).strftime('%I:%M%p%Z UTC on %m/%d/%Y'))
```
[the remark this answers is not in the document]
Comment (nathan-weinberg) [reply scripts/nope.py:419]: Yup, would change timestamp display to show localized time, which is EST locally for us and UTC for Jenkins. Now it should be consistent

File: scripts/flows_v1alpha1_flowcollector_lokistack.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (memodi) [reply scripts/flows_v1alpha1_flowcollector_lokistack.yaml:25]: just this one. This is newly added.

**Comment by memodi** [comment]
> New installation steps can be seen succeeding here: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/232/

@nathan-weinberg - does this PR address where user may already have NetObserv but wishes to update some params? if so, could you have that as a success run as well? thanks

**Comment by memodi** [comment]
> Sure, do you have a cluster I can use?
unfortunately not the Flexy cluster

**Comment by nathan-weinberg** [comment]
> > Sure, do you have a cluster I can use?
> 
> unfortunately not the Flexy cluster

Flexy is disabled, so we probably would not be able to do this until Monday

**Comment by memodi** [comment]
> Flexy is disabled, so we probably would not be able to do this until Monday

I was able to get one created luckily, Flexy id: 155078 , let me know when you're done there!

**Comment by memodi** [comment]
> I was able to get one created luckily, Flexy id: 155078 , let me know when you're done there!
actually this is the incorrect one, I am building one now since Jenkins is back now.

**Comment by nathan-weinberg** [comment]
Unfortunately we are still seeing issues with Flexy - I will update this PR will successful test jobs ASAP

**Comment by nathan-weinberg** [comment]
Successful Jobs:
- Initial Install: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/239/
- Updated Config: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/240/

**Pull request merged**